modules/SofaGraphComponent/SceneCheckAPIChanges.cpp

```diff
@@ -110,8 +113,11 @@ void SceneCheckAPIChange::doCheckOn(Node* node)
 void SceneCheckAPIChange::installDefaultChangeSets()
 {
     addHookInChangeSet("17.06", [](Base* o){
-        if(o->getClassName() == "RestShapeSpringsForceField" && o->findData("external_rest_shape")->isSet())
-            msg_warning(o) << "RestShapeSpringsForceField have changed since 17.06. The parameter 'external_rest_shape' is now a Link. To fix your scene you need to add and '@' in front of the provided path. See PR#315" ;
+        if(o->getClassName() == "RestShapeSpringsForceField" && o->findLink("external_rest_shape") )
+        {
+            if( o->findLink("external_rest_shape")->getSize() != 0)
```

Comment:
What is `getSize() != 0` supposed to check? 
If a `RestShapeSpringsForceField` is implemented with a wrong `external_rest_shape`, the link will be created but empty. If a `RestShapeSpringsForceField` is implemented with NO `external_rest_shape`, the link will still be created and empty.
I I feel like there is no way to differentiate a wrong Link from an empty Link after component instantiation.